util/feature.txt

```diff
@@ -0,0 +1,5 @@
+feature 1 implementation
+
+ feature 2 implementation
```

Comment:
"Let's add a bit more to the feature 2 implementation. I think it should handle multiple widgits.

Reply:
@othmanm7mad see the latest commit for the change